app/views/api_docs/candidate_api_docs/reference/reference.html.erb

```diff
@@ -0,0 +1,59 @@
+<h1 class="govuk-heading-xl"><%= t('page_titles.api_docs.candidate_api.reference') %></h1>
+
+<p class="govuk-body">This is API documentation for the Apply for teacher training candidates API. This API provides candidate information to other services.</p>
```

Comment:
Let's nail this down now: is this the Candidate API or the Candidate**s** API?

Reply:
Sorry @duncanjbrown I should have popped a note on the PR when I made the change. JAmes and I, decided to keep things uniform that we should make it the `Candidate API`. ☝️ this is just one I missed, all sorted now.

Reply:
Great 🎉